src/ax_interface/mib.py

```diff
@@ -30,17 +30,29 @@ def __init__(self):
 
     async def start(self):
         # Run the update while we are allowed
+        redis_exception_happen = False
         while self.run_event.is_set():
             try:
                 # reinit internal structures
                 if self.update_counter > self.reinit_rate:
+                    # reconnect when redis exception happen
+                    if redis_exception_happen:
+                            self.reinit_connection()
```

Comment:
>     [](http://example.com/codeflow?start=24&length=4)

You can remove one extra level of indentation.

Reply:
Fixed